openml/flows/sklearn_converter.py

```diff
@@ -625,28 +625,33 @@ def _serialize_cross_validator(o):
 
 
 def _check_n_jobs(model):
-    '''
+    """
     Returns True if the parameter settings of model are chosen s.t. the model
-     will run on a single core (in that case, openml-python can measure runtimes)
-    '''
-    def check(param_dict, disallow_parameter=False):
-        for param, value in param_dict.items():
-            # n_jobs is scikitlearn parameter for paralizing jobs
-            if param.split('__')[-1] == 'n_jobs':
-                # 0 = illegal value (?), 1 = use one core,  n = use n cores
-                # -1 = use all available cores -> this makes it hard to
-                # measure runtime in a fair way
-                if value != 1 or disallow_parameter:
+    will run on a single core (in that case, openml-python can measure runtimes)
```

Comment:
as well as the function documentation?

Reply:
Same. Function description is still up to date.